docs/victoriametrics/stream-aggregation/configuration.md

```diff
@@ -184,6 +184,8 @@ support the following approaches for hot reloading stream aggregation configs fr
   ```
 
 * By sending HTTP request to `/-/reload` endpoint (e.g. `http://vmagent:8429/-/reload` or `http://victoria-metrics:8428/-/reload`).
+* Use the `-remoteWrite.configCheckInterval` {{% available_from "#" %}} flag to control how often `vmagent` reloads configs specified via `-remoteWrite.streamAggr.config`, `-streamAggr.config`, `-remoteWrite.relabelConfig` and `remoteWrite.urlRelabelConfig` flags. Reload is disabled by default.
```

Comment:
this is consufing. why `remoteWrite.configCheckInterval` reloads `-streamAggr.config`? should it be controlled via `-streamAggr.configCheckInterval`?

Reply:
To me it would seem less confusing if it will be controlled by separate flags.